elements/californium.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: Californium
+---
+
+Symbol : Cf
+Atomic Number : 98
+
+Californium was first synthesized in 1950 at Lawrence Berkeley National Laboratory by bombarding curium with alpha particles. 
+
+Fun Facts:
+- It was first synthesized in 1950 at Lawrence Berkeley National Laboratory! 
+- It was named after the state of California and the University of California, Berkeley
+- It has the second-highest atomic mass of all elements that can be seen with the naked eye. 
+- It has 20 known isotopes, with californium-251 being the most stable and having a half-life of 898 years. 
+- It has a melting point of 1,652°F and slowly tarnishes in air at room temperature. 
```

Comment:
What is this °F symbol?? i have never seen it before and feel a shiver down my spine at it's sight. Oh how I miss the vast oceanic land of °C

Reply:
I believe °F is an American term, short for "Fuck you metric system!". I'll update to include celsius as well fahrenheit, to be more inclusive